incubator/hnc/internal/reconcilers/object.go

```diff
@@ -763,7 +764,13 @@ func (r *ObjectReconciler) syncUnpropagatedSource(ctx context.Context, log logr.
 // - Objects of a type whose mode is set to "remove" in the HNCConfiguration singleton
 // - Objects with nonempty finalizer list
 // - Service Account token secrets
-func (r *ObjectReconciler) shouldPropagateSource(log logr.Logger, inst *unstructured.Unstructured) bool {
+func (r *ObjectReconciler) shouldPropagateSource(log logr.Logger, inst, destInst *unstructured.Unstructured) bool {
```

Comment:
The only information from `destInst` we're using here is its namespace name. It's a good idea to minimize the amount of information that you pass around the system, so I'd suggest just passing in the destination namespace name as a string.

Also very minor nit - we usually use `dst` instead of `dest` because it lines up nicely with `src`. I'd either call the string argument `dst` or `dstNS` but I think `dst` is clear enough from the context.

Reply:
done